types/jsonld/jsonld-spec.d.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Types from the jsonld Specification:
+ * https://www.w3.org/TR/json-ld-api/
+ *
+ */
+
+// Some aliases and placeholders for better readability
+export type JsonLdObj = object;
+export type JsonLdArray = [object];
+export type JsonLd = JsonLdObj|JsonLdArray;
+
+type DOMString = string;
+type LoadDocumentCallback = Promise <Url>;
```

Comment:
Remove space between Identifier and Type Argument list

Reply:
Fixed